extensions/barrage/src/main/java/io/deephaven/extensions/barrage/chunk/ChunkInputStreamGenerator.java

```diff
@@ -204,8 +205,18 @@ static WritableChunk<Values> extractChunkFromInputStream(
                         Double.BYTES, options,fieldNodeIter, bufferInfoIter, is, outChunk, outOffset, totalRows);
             case Object:
                 if (type.isArray()) {
-                   return VarListChunkInputStreamGenerator.extractChunkFromInputStream(
-                           options, type, fieldNodeIter, bufferInfoIter, is, outChunk, outOffset, totalRows);
+                    if(componentType == byte.class) {
```

Comment:
The style here is slightly off. If you run `./gradlew spotlessApply` at the top level, then everything non-comforming will be formatted automatically. (Otherwise, once CI runs, it will fail here and in a few other locations.)

Reply:
should now be spotless